### DISCUSSION TITLE
How to disable the validation when opening the calendar?

### DISCUSSION THREAD

**Discussion by romanh24** [wojtekmaj/react-daterange-picker#168] [Q&A] [answered]
The red color is displayed when the calendar is opened. I understand that this is validation. How to turn it off?
Thanks for the answer.

![image](https://user-images.githubusercontent.com/70602437/198018783-4e52d1a9-711e-4763-b22f-e49ec4f06860.png)

**Comment by wojtekmaj** [comment, marked as answer]
It's just styling of `input:invalid`. Put `background: none` on it and you're done.
